redhat/rdma-core.spec

```diff
@@ -345,39 +345,38 @@ install -D -m0644 ibacm_opts.cfg %{buildroot}%{_sysconfdir}/rdma/
 rm -rf %{buildroot}/%{_initrddir}/
 rm -rf %{buildroot}/%{_sbindir}/srp_daemon.sh
 
-# infiniband-diags
+%post -n rdma-core
+# we ship udev rules, so trigger an update.
+/sbin/udevadm trigger --subsystem-match=infiniband --action=change || true
+/sbin/udevadm trigger --subsystem-match=net --action=change || true
+/sbin/udevadm trigger --subsystem-match=infiniband_mad --action=change || true
+
 %post -n infiniband-diags -p /sbin/ldconfig
 %postun -n infiniband-diags -p /sbin/ldconfig
 
-# libibverbs
```

Comment:
So strange that comments cause things to fail, are you sure? Seems nearly impossible.. No objection to removing them though

Reply:
Yes, I am sure.

$ rpm -qp --scripts librdmacm-26.0-6.el8.x86_64.rpm
postinstall program: /sbin/ldconfig
postuninstall scriptlet (using /bin/sh):


# ibacm
^
Because of the heading '#', 'git commit' ignored the last line.